sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -789,17 +790,62 @@
                 "featureStoreSettings": {
                     "computeruntime": {
                         "SparkRuntimeVersion": "[parameters('spark_runtime_version')]"
-                    },
-                    "offlinestoreconnectionname": "[parameters('offline_store_connection_name')]",
-                    "onlinestoreconnectionname": "[parameters('online_store_connection_name')]"
+                    }
                 },
                 "enableDataIsolation": "[parameters('enable_data_isolation')]",
                 "workspaceHubConfig": "[parameters('workspace_hub_config')]",
                 "existingWorkspaces": "[parameters('existing_workspaces')]",
                 "azureOpenAI": {
                     "resourceId": "[parameters('byo_open_ai_resource_id')]"
                 }
-            }
+            },
+            "resources":[
+                {
+                    "condition": "[equals(parameters('set_up_feature_store'), 'true')]",
+                    "type": "connections",
+                    "apiVersion": "2022-05-01",
+                    "name": "[variables('offlineStoreConnectionName')]",
+                    "location": "[parameters('location')]",
+                    "dependsOn": [
+                        "[resourceId('Microsoft.MachineLearningServices/workspaces', parameters('workspaceName'))]",
+                        "[variables('offlineStoreConnectionTarget')]"
+                    ],
+                    "identity": {
+                        "type": "SystemAssigned"
```

Comment:
why this is SystemAssigned? (same for the following for online connection).

Reply:
workspace connection needs a system assigned identity, this is from the original arm template.

Reply:
identity here can be systemAssigned OR  userAssigned OR  systemAssigned,userAssigned. we support all of them

Reply:
Do we use the connection identity anywhere? I thought this was just a requirement since connection is considered a sub-resource, but we are not going to use any identity created on it so probably SystemAssigned is enough.